cmd/crane/cmd/root.go

```diff
@@ -67,12 +68,26 @@ func New(use, short string, options []crane.Option) *cobra.Command {
 			cf, err := config.Load(os.Getenv("DOCKER_CONFIG"))
 			if err != nil {
 				logs.Debug.Printf("failed to read config file: %v", err)
+				return err
 			} else if len(cf.HTTPHeaders) != 0 {
 				options = append(options, crane.WithTransport(&headerTransport{
-					inner:       http.DefaultTransport,
+					inner: &http.Transport{
+						TLSClientConfig: &tls.Config{InsecureSkipVerify: insecure},
```

Comment:
I'd dedupe this a bit, something like:

```
transport := http.DefaultTransport.(*http.Transport).Clone()
transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: insecure}

...
} else if len(cf.HTTPHeaders) != 0 {
  transport = &headerTransport{transport, cf.HTTPHeaders}
}

options = append(options, crane.WithTransport(transport))

Reply:
I applied some changes if I try your suggestion or something related I get this error
`cannot use &(headerTransport literal) (value of type *headerTransport) as *http.Transport value in assignment`

with my limited knowledge, I don't know how to solve it.

Reply:
thanks for your review and feedback

Reply:
> cannot use &(headerTransport literal) (value of type *headerTransport) as *http.Transport value in assignment

Ah, yeah i was just using some shorthand, what you did instead works 👍